main.lua

```diff
@@ -20,9 +20,12 @@ function love.load()
     require("class.grid")
     require("class.tower")
     require("class.enemy")
+    require("class.round")
     require("class.world")
     require("towers.saw") 
     require("enemies.smallguy")
+    require("enemies.largeguy")
+    rounds = require("rounds")
```

Comment:
this 'rounds' variable is declared as a global variable. Meaning ANY of these scripts have access to it, which is pretty dangerous/error-prone. Prefix this with `local` and then pass it as a parameter to the  World() constructor below